values.yaml

```diff
@@ -105,10 +105,12 @@ serviceAccount:
     name: 
 
 aws:  
-  awsMarketplace: ${useAWSMarketplace}
```

Comment:
Is this name change for variable correct?

Reply:
I have fixed this mismatch on Helm chart repo